babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -1233,6 +1233,24 @@ private void checkParseInfixCast(String sqlType) {
         + "RETURN (CURRENT_DATE + 1)";
     sql(sql).ok(expected);
   }
+  @Test public void testTranslateUsingCharacterSet() {
+    expr("translate ('abc' using latin_to_unicode)")
+        .ok("TRANSLATE ('abc' USING `LATIN`_TO_`UNICODE`)");
```

Comment:
`LATIN`_TO_`UNICODE` looks a little strange with the "`" in the middle. Is this intended?

Reply:
This is intended, this is a design decision made on spot. I imagine having the "`" would make it easier for the future use, while I am not sure what is the requirement for unparsing. I will add a section in the design doc for discussion.

Reply:
It's usually a good idea to explicitly call out behavior/design decision in the design doc rather than make them on the spot while coding.

Thanks for adding a section to the doc now and we can continue the conversation on this there.

Reply:
Fix in https://github.com/googleinterns/calcite/pull/84/commits/ca862f5ff21ff88a9d4fe57b97da239c54799f3f